index.bs

```diff
@@ -611,8 +607,8 @@ It must act as follows:
 <h4 id="close-method">
 The close method</h4>
 
-The <dfn id="dfn-close" method for=Blob>close()</dfn> method is said to <a lt="closed" for=Blob>close</a> a {{Blob}},
-and must act as follows on the {{Blob}} on which the method has been called:
+The <dfn id="dfn-close" method for=Blob>close()</dfn> method is said to <a lt="closed" for=Blob>close</a>
+the {{Blob}} context object when it acts on the {{Blob}} as follows:
```

Comment:
I don't really like this rephrasing, in particular using "when" to combine the two sentence fragments sounds wrong to me (but I'm not a native english speaker). I'd rather just get rid of the whole "on the {{Blob}} on which the method has been called" part, but otherwise keep the current text.